sites/common/includes/base/tab-navigation-bar.html

```diff
@@ -1,40 +1,33 @@
 <div id="tab-navigation-bar">
   <div class="tab-navigation-links">
-    <a href="https://www.thunderbird.net">Thunderbird</a>
-    <a href="https://www.tb.pro" class="active">Thunderbird Pro</a>
+    <a href="{{ settings.CANONICAL_URL }}" data-stage-url="{{ settings.CANONICAL_STAGE_URL }}">Thunderbird</a>
+    <a href="{{ settings.CANONICAL_TBPRO_URL }}" class="active" data-stage-url="{{ settings.CANONICAL_TBPRO_STAGE_URL}}">Thunderbird Pro</a>
```

Comment:
Decided to do away with looping and matching. Realized that if we tag each link with an ID, might as well just tag each one with the eventual URL to swap.